pkg/aggregator/resourcetagger/resource_tagger.go

```diff
@@ -156,9 +172,10 @@ func (d *tagDecorator) Decorate(ctx context.Context, object util.ObjectRef, even
 	for tagKey, jsonPath := range d.resourceTagMapping[gr] {
 		val, err := getValFromJsonPath(raw, jsonPath)
 		if err != nil {
-			return
+			continue
 		}
-		event.Tags[tagKey] = val
+		tagMetric.Result = "Success"
```

Comment:
So if all tag paths are not successful, we will report the Unknown metric here, while if at least one tag path is successful, we log with metric Success?